pysat/instruments/icon_ivm.py

```diff
@@ -293,66 +215,29 @@ def clean(inst, clean_level=None):
     """
 
     if clean_level != 'none':
```

Comment:
We can remove all these checks. The cleaning method is only invoked if clean_level is not None.